sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -0,0 +1,3270 @@
+        "operationId": "RegistryManager_QueryIotHub",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "querySpecification",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QuerySpecification"
+            },
+            "description": "The query string to run. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "x-ms-continuation",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "The continuation token to get the next page of results."
+          },
+          {
+            "name": "x-ms-max-item-count",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "The maximum number of items to return per page. The service may use a different value if the value specified is not acceptable."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Query result with continuation token if appropriate.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Twin"
+              }
+            },
+            "headers": {
+              "x-ms-item-type": {
+                "description": "Type of the list of items.",
+                "type": "string"
+              },
+              "x-ms-continuation": {
+                "description": "Continuation token",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language."
+      }
+    },
+    "/devices/{id}": {
+      "get": {
+        "description": "Retrieve a device from the identity registry of an IoT hub. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "operationId": "RegistryManager_GetDevice",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Device ID.",
```

Comment:
Device Id?

Reply:
This doesn't look as descriptive as I would expect. I'll fix it